### DISCUSSION TITLE
How to boot windows images of qcow2?

### DISCUSSION THREAD

**Discussion by maocili** [cloud-hypervisor/cloud-hypervisor#3875] [Q&A]
I am try to boot windows images of qcow2. what should I do if I want boot it?

```
sudo qemu-system-x86_64 \
        -machine q35,accel=kvm \
        -cpu host -m 2G \
        -bios $OVMF_DIR/OVMF.fd \
        -cdrom $WIN_ISO_FILE \
        -drive file=$VIRTIO_ISO_FILE,index=0,media=cdrom \
        -drive if=none,id=root,file=$IMG_FILE  \
        -device virtio-blk-pci,drive=root,disable-legacy=on  \
        -device virtio-net-pci,netdev=mynet0,disable-legacy=on  \
        -netdev user,id=mynet0  \
        -vga std \
        -enable-kvm


```

```
/root/cloud-hypervisor/cloud-hypervisor/target/release/cloud-hypervisor \
	--kernel $OVMF_DIR/CLOUDHV.fd \
	--disk path=./$IMG_FILE \
	--cpus boot=2,kvm_hyperv=on \
	--memory size=1G \
	--serial tty \
	--console off \
	--net tap=
```

**Comment by rbradford** [comment]
Did you follow the instructions in [docs/windows.md](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/windows.md)?

**Reply by maocili** [reply]
> Did you follow the instructions in [docs/windows.md](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/windows.md)?

yes. I can boot with raw

**Reply by rbradford** [reply]
Good.  Our QCOW support does not support compressed blocks which it is possible you were using. You would have received an error like: https://github.com/cloud-hypervisor/cloud-hypervisor/issues/3727

Raw file are recommended for performance and compatibility.
